bigquery/bigquery_quickstart/main.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START bigquery_quickstart]
+// Sample bigquery_quickstart creates a Google BigQuery dataset.
+package main
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
+	"log"
+
+	// Imports the Google Cloud Datastore client package
```

Comment:
We follow the typical Go comment style to keep these types of guidance comments as proper sentences.

// Imports the Google Cloud Datastore client package.